### DISCUSSION TITLE
August Community Member Shoutout - im-coder-lg

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5424] [General]
This month, we want to take a second to thank @im-coder-lg for all their work answering [discussions](https://github.com/coder/code-server/discussions?discussions_q=im-coder-lg), responding to issues and be generally kind helping others in our community. You inspire the rest of us to do the same! 

From everyone on the code-server team & Coder, we appreciate you! 🙏🏼 🙌🏼

**Comment by lg-nineoneninethreenine** [comment]
OMG thanks! ❤️
